### DISCUSSION TITLE
请问按量收费是怎么收费的，里面有好多功能是要收费的，能不能分别举个例子。

### DISCUSSION THREAD

**Discussion by loocheeming** [ZuodaoTech/everyone-can-use-english#566] [Q&A]
我账户上有1美元，我只是对一篇文章提取关键字，就扣了1半的费用，这个到底怎么收费的。能不能有了收费明细。

**Comment by an-lee** [comment]
都是调用 OpenAI 的费用，可以参考这个 https://openai.com/pricing

文章的话，除了提取关键词，还会自动对每个词进行翻译，文章比较长的话，消耗的 token 就会比较多。

**Reply by an-lee** [reply]
后续会提供一些其他更便宜的模型供选择，目前默认都是使用 OpenAI GPT-4-turbo 模型。

如果你有 OpenAI 的 key，可以自己配置，这样就不需要扣余额了（除了发音评估功能）。
